js/ez-navigation-hub.js

```diff
@@ -1,5 +1,33 @@
 (function () {
-    /** @polymerElement */
+    /**
+    * <ez-navigation-hub> represents the navigation hub of the application.
+    * Zones can be selected to display the links refering to the different parts of the application.
+    *
+    * Each zone of the navigation hub needs a data-zone-identifier.
```

Comment:
Zones in the navigation hub are identified by a `data-zone-identifier` containing the zone identifier.